content/ಅಂಕಿಗಳ ವಿಭಕ್ತಿ.md

```diff
@@ -0,0 +1,97 @@
+# ಕನ್ನಡದ ಸಂಖ್ಯೆಗಳಲ್ಲಿ ಅಡಗಿರುವ ವಿಭಕ್ತಿಗಳು
+
```

Comment:
```suggestion
## ಪರಿವಿಡಿ

- [ಕನ್ನಡದ ಸಂಖ್ಯೆಗಳಲ್ಲಿ ಅಡಗಿರುವ ವಿಭಕ್ತಿಗಳು](#ಕನ್ನಡದ-ಸಂಖ್ಯೆಗಳಲ್ಲಿ-ಅಡಗಿರುವ-ವಿಭಕ್ತಿಗಳು)
- [ಪರಿವಿಡಿ](#ಪರಿವಿಡಿ)
- [ಪ್ರಸ್ತಾವನೆ](#ಪ್ರಸ್ತಾವನೆ)
- [ಪರವಾನಗಿ / License](#ಪರವಾನಗಿ--license)
- [ಹದಿಮೂರು](#ಹದಿಮೂರು)
  - [ಹತ್ತು](#ಹತ್ತು)
  - [ತೃತಿಯೆಯೂ ಪಂಚಮಿಯೂ](#ತೃತಿಯೆಯೂ-ಪಂಚಮಿಯೂ)
  - [ಪತ್ತಿಂ ಮೂರು](#ಪತ್ತಿಂ-ಮೂರು)
  - [ಪದಿಂ, ಪಯಿಂ, ಪನ್](#ಪದಿಂ-ಪಯಿಂ-ಪನ್)
- [ಲಗಂ ವರ್ಜನೆ](#ಲಗಂ-ವರ್ಜನೆ)
- [ಹನ್ನೊಂದು](#ಹನ್ನೊಂದು)
- [ಹನ್ನೆರಡು](#ಹನ್ನೆರಡು)
- [ಹದಿಮೂರು](#ಹದಿಮೂರು-1)
- [ಹದಿನಾಲ್ಕು](#ಹದಿನಾಲ್ಕು)
- [ಹದಿನೈದು](#ಹದಿನೈದು)
- [ಹದಿನಾರು](#ಹದಿನಾರು)
- [ಹದಿನೇಳು](#ಹದಿನೇಳು)
- [ಹದಿನೆಂಟು](#ಹದಿನೆಂಟು)
- [ಸಮಾರೋಪ](#ಸಮಾರೋಪ)

```

You can update it by running `cat content/ಅಂಕಿಗಳ\ ವಿಭಕ್ತಿ.md | ./hack/print-article-content.sh | pbcopy` on the repo root folder and pasting the contents under `## ಪರಿವಿಡಿ`